src/block-kit/composition/Mrkdwn.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx JSXSlack.h */
+import { JSXSlack } from '../../jsx'
+import { ObjectOutput } from '../../utils'
+import html from '../../html'
+import { BlockComponentProps } from '../Blocks'
+
+interface MrkdwnProps extends BlockComponentProps {
+  children: JSXSlack.Children<BlockComponentProps>
```

Comment:
`<Mrkdwn>` component would not take block components as children. It would be better to take `JSXSlack.Children<{}>` as same as children in `<Section>` and `<Context>`.